### ISSUE TITLE
A Better Progress Bar

### ISSUE THREAD

**Issue by hanlint** [mosaicml/composer#347]
cc: @siriuslee @A-Jacobson @Landanjs

**Comment by ravi-mosaicml** [comment]
Can you elaborate?

**Comment by A-Jacobson** [comment]
I'll do my best.

TLDR: the progress bar should at least tell us total training time and model metrics at a glance.

our current progress bar looks like this:
<img width="658" alt="Screen Shot 2022-02-11 at 1 36 56 AM" src="https://user-images.githubusercontent.com/9411532/153568483-7403c4be-fd78-4b56-bb86-13f4fb10b10f.png">

There are a few things lacking with this setup:
1. It could use an overall progress bar so the user can get an estimate of total training time without doing some mental math.

2. It doesn't log enough metrics, I had a hard time logging anything except validation accuracy. Which isn't really enough to tell if your model is improving.

the fastai progress bar is sort of the extreme example of this, they render tables of training metrics as they train.
<img width="484" alt="Screen Shot 2022-02-11 at 1 51 13 AM" src="https://user-images.githubusercontent.com/9411532/153570538-6998d223-0fc4-4e5a-b09f-d8ad8d6caacf.png">

Their library can even render plots during training though i dont know if we need to go that far.
https://github.com/fastai/fastprogress

**Comment by Landanjs** [comment]
Austin provided great points for improvement! 

For me specifically, displaying the learning rate would be helpful to debug if it is increasing/decreasing at the expected times. Likely, it would be helpful to visualize most callback monitored metrics on the progress bar. Providing visualizations/table are bonuses for improving debugging

**Comment by mvpatel2000** [comment]
Closing. Tracking elsewhere as low pri
